Lib/ufo2ft/featureWriters/kernFeatureWriter.py

```diff
@@ -726,13 +729,17 @@ def _makeSplitScriptKernLookups(self, lookups, pairs, ignoreMarks=True, suffix="
                         pair.value,
                     )
                     continue
-                scriptIsRtl = script_horizontal_direction(script, "LTR") == "RTL"
+                directions = {script_direction(script) for script in scripts}
+                assert len(directions) == 1
+                scriptIsRtl = directions == {"RTL"}
                 # Numbers are always shaped LTR even in RTL scripts:
                 pairIsRtl = scriptIsRtl and "L" not in bidiTypes
                 rule = self._makePairPosRule(
                     pair, side1Classes, side2Classes, pairIsRtl
                 )
                 lookup.statements.append(rule)
+            for script in scripts:
+                lookups.setdefault(script, {})[lookupName] = lookup
```

Comment:
why do we need to add the same `lookup` (with same `lookupName`) multiple times for each script in `lookups` dict (and then later on filter these duplicates out)? Can't we simply change `lookups` dict to be keyed by the lookupName instead of a dict of dicts?

Reply:
i guess we still use that `script` key elsewhere (to know if it's a default kind of lookup or dist-enabled), ignore my previous comment then.